domains/uzi.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "crischutu07",
+           "email": "",
+           "discord": "1001747794954039368"
+        },
+    
+        "record": {
+            "CNAME": "crischutu07.github.io/uzi"
```

```suggestion
            "CNAME": "crischutu07.github.io"
```
CNAMEs cannot contain slashes `/`.